pandas/core/frame.py

```diff
@@ -4969,6 +4969,39 @@ def f(x):
 
         return Series(result, index=labels)
 
+    def nunique(self, axis=0, dropna=True):
+        """
+        Return Series with number of distinct observations over requested
+        axis.
+
+        .. versionadded:: 0.20.0
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            0 or 'index' for row-wise, 1 or 'columns' for column-wise
```

Comment:
As is, the wording is consistent with all the other methods such as `count()`: Wouldn't it be better to have a dedicated PR for that, in case all the `axis` docstrings are to be improved?